internal/matchers/archive.go

```diff
@@ -84,3 +84,10 @@ func Rar(in []byte) bool {
 func Warc(in []byte) bool {
 	return bytes.HasPrefix(in, []byte("WARC/"))
 }
+
+// Zstd matches a Zstandard archive file
```

Comment:
Comments are exported to docs and they should be complete sentences, including the ending period.